packages/docusaurus-theme-classic/src/theme/Layout/index.tsx

```diff
@@ -13,6 +13,7 @@ import {
   ThemeClassNames,
   useKeyboardNavigation,
 } from '@docusaurus/theme-common';
+import useMermaid from '@docusaurus/theme-mermaid/theme/useMermaid';
```

Comment:
This should be removed—see my other comments. The classic theme should not know about the existence of the mermaid theme.

Reply:
Done